### PR TITLE
Authorise incineration status access

### LINKED ISSUE #44: Authorise incineration status access with incineration_status_emails

### PR DESCRIPTION
**Pull request by shreyansh-g24**
FIXES #44

### CODE REVIEW

**Comment by shreyansh-g24** [comment]
@unnitallman _a Please check

File: test/dummy/app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::Base
+  def current_user
```
Comment (unnitallman) [inline test/dummy/app/controllers/application_controller.rb:2]: Why is this needed? This could override the host app's `current_user` method right?

Comment (shreyansh-g24) [reply test/dummy/app/controllers/application_controller.rb:2]: This method is in the test dummy application for testing.

**Review by unnitallman — APPROVED** [review]
